care/facility/api/serializers/patient.py

```diff
@@ -159,6 +159,9 @@ class PatientDetailSerializer(PatientListSerializer):
     class MedicalHistorySerializer(serializers.Serializer):
         disease = ChoiceField(choices=DISEASE_CHOICES)
         details = serializers.CharField(required=False, allow_blank=True)
+        status = serializers.CharField(required=False, allow_blank=True)
+        duration = serializers.CharField(required=False, allow_blank=True)
+        type  = serializers.CharField(required=False, allow_blank=True)
```

Comment:
you can add validation for type based on the disease

Reply:
@sainak Can you give me a pointer on how/where to add validation

Reply:
@AshrafMd-1 you can extend `validate` method provided by the Serializer class.

Refer: https://www.django-rest-framework.org/api-guide/serializers/#object-level-validation